price-feeder/tests/integration/provider_test.go

```diff
@@ -43,6 +43,23 @@ func (s *IntegrationTestSuite) TestWebsocketProviders() {
 	for key, pairs := range cfg.ProviderPairs() {
 		providerName := key
 		currencyPairs := pairs
+
+		// Filter by provider
+		// if providerName != provider.ProviderBitget {
+		// 	continue
+		// }
+
+		// Filter by currency pair
+		// currencyPairs = []types.CurrencyPair{}
+		// for _, pair := range pairs {
+		// 	if pair.Base == "MATIC" {
+		// 		currencyPairs = append(currencyPairs, pair)
+		// 	}
+		// }
+		// if len(currencyPairs) == 0 {
+		// 	continue
+		// }
```

Comment:
Let's take out commented code before merging

Reply:
ok :( I'll find a cleaner way to add some options to this later.

Reply:
<3